packages/fx-core/resource/package.nls.json

```diff
@@ -1175,10 +1175,13 @@
   "driver.deploy.error.unknownError": "Unknown error happened: %s",
   "driver.deploy.notice.deployAcceleration": "Deploying to Azure App Service takes a long time. Consider referring to this document to optimize your deployment:",
   "driver.deploy.error.noFileFindInDistributionFolder": "No file is found in distribution folder. Please check if the distribution folder is empty or you have ignored everything.",
-  "driver.deploy.notice.deployDryRunComplete": "Preparations of deployment are complete. ",
+  "driver.deploy.notice.deployDryRunComplete": "Preparations of deployment are complete. You can find the package in the `%s`",
   "driver.deploy.azureAppServiceDeploySummary": "Project deployed to Azure App Service successfully.",
   "driver.deploy.azureFunctionsDeploySummary": "Project deployed to Azure Functions successfully.",
   "driver.deploy.azureStorageDeploySummary": "Project deployed to Azure Storage successfully.",
+  "driver.deploy.azureAppServiceDeployDetailSummary": "Deploy `%s` to Azure App Service successful.",
```

Comment:
Deploy -> Deployed

successful -> successfully

apply to the lines after this as well.

Reply:
changed to another diction.